eox_tagging/test/integration/test_api_integration.py

```diff
@@ -0,0 +1,173 @@
+"""
+Integration tests.
+
+This suite performs multiple http requests to guarantee
+that the eox-tagging rest API is behaving as expected on a live server.
+"""
+import json
+from os import environ
+from unittest import skipUnless
+
+import requests
+from django.test import SimpleTestCase
+from rest_framework import status
+
+
+@skipUnless(environ.get("TEST_INTEGRATION"), "integration")
+class TagTestCase(SimpleTestCase):  # pragma: no cover
+    """Test suite"""
+
+    @classmethod
+    def setUpClass(cls):
+        """
+        Initialize data fixtures
+        """
+
+        with open("eox_tagging/test/integration/test_data.json") as file:
+            cls.data = json.load(file)
+
+        base_url = cls.data["base_url"]
+        endpoint = "/eox-tagging/api/v1/tags/"
+
+        cls.url = "{}{}".format(base_url, endpoint)
+        cls.session = requests.Session()
+        cls.session.headers.update(
+            {
+                "Authorization": "Bearer {}".format(cls.data["access_token"]),
```

Comment:
this will fail if access_token doesn't exist

Reply:
But the token should always exist because you have to setup that alongside the other data beforehand.